starter-code/public/index.html

```diff
@@ -85,9 +85,11 @@
     <script src="vendor/scripts/highlight.pack.js"></script>
     <script src="/scripts/article.js"></script>
     <script src="/scripts/articleView.js"></script>
-    <!-- TODO: Due to changes in article.js and articleView.js to attach their interior objects to a global 'app' object, we need to make changes to the contents of the following <script> tag: -->
+    <!-- DONE: Due to changes in article.js and articleView.js to attach their interior objects to a global 'app' object, we need to make changes to the contents of the following <script> tag: -->
+    <!-- Estimated 5 min | Actual 2 min -->
     <script>
-      Article.fetchAll(articleView.initIndexPage);
+      articleView.initIndexPage();
```

Comment:
Your index is not rendering and I am getting this issue in the console: Uncaught ReferenceError: Article is not defined
    at (index):92